fixtures/ext-types/http-headermap/src/lib.rs

```diff
@@ -3,7 +3,7 @@ use std::str::FromStr;
 pub(crate) use http::HeaderMap;
 
 /// A single HttpHeader for Uniffi bindings
-pub(crate) struct HttpHeader {
+pub struct HttpHeader {
```

Comment:
This is because of `can't leak crate-private type` on the use of `uniffi::custom_type`.

I think it was an anomaly that a `pub(crate)` type could be used within a custom type (e.g. here it is `sequence<HttpHeader>` in the UDL ) before so I didn't investigate what actually caused it.